cfgov/unprocessed/css/molecules/info-unit.less

```diff
@@ -71,91 +129,39 @@
     }
 }
 
-.content_main__home {
-    padding-top: unit( @grid_gutter-width / @base-font-size-px, em );
-
-    .respond-to-max( @bp-xs-max, {
-        padding-bottom: unit( ( @grid_gutter-width * 2 ) / @base-font-size-px, em );
-    } );
-}
-
-.content-l_col__home {
-    .respond-to-max( @bp-med-max, {
-        + .content-l_col__home {
-            margin-top: unit( ( @grid_gutter-width / 2 ) / @base-font-size-px, em );
-        }
-    } );
-
-    .respond-to-min( @bp-sm-min, {
-        .grid_column( 1, 1 );
-    } );
-
-    .respond-to-min( @bp-lg-min, {
-        .grid_column( 4 );
-    } );
-}
-
-.o-well__home {
-    .respond-to-min( @bp-xl-min, {
-        max-width: 1230px - ( 45px * 2 ) !important;
-        margin: 0 auto;
-        padding-right: unit( 45px / @base-font-size-px, em ) !important;
-        padding-left: unit( 45px / @base-font-size-px, em ) !important;
-    } );
-}
+/*
+    Inline info-unit Helper Mixin -
+    This mixin is used to modify an info-unit to an inline arrangement.
+    Most of the time the `.m-info-unit__inline` class will be sufficient,
+    but there are limited situations the inline arrangement needs to be conditioned.
+    See /pages/home.less L:103
+ */
+.u-info-unit-base__inline( @img_size ) {
+    .u-clearfix();
+
+    .m-info-unit_image {
```

Comment:
Yes, because it's always used for the inline modifier, no matter what the image size is.